pkg/operations/kubernetesupgrade/upgradeagentnode.go

```diff
@@ -38,13 +38,20 @@ type UpgradeAgentNode struct {
 // backs up/preserves state as needed by a specific version of Kubernetes and then deletes
 // the node
 func (kan *UpgradeAgentNode) DeleteNode(vmName *string) error {
+	var kubeAPIServerURL string
+
+	if kan.UpgradeContainerService.Properties.HostedMasterProfile != nil {
+		kubeAPIServerURL = kan.UpgradeContainerService.Properties.HostedMasterProfile.FQDN
+	} else {
+		kubeAPIServerURL = kan.UpgradeContainerService.Properties.MasterProfile.FQDN
+	}
 
 	// Currently in a single node cluster the api server will not be running when this point is reached on the first node so it will always fail.
```

Comment:
because of this can we set kubeconfig to nil in single master upgrade command? or fail with a force option that makes kubeConfig empty/nil?